mets_mods2tei/api/tei.py

```diff
@@ -588,9 +588,9 @@ def add_collection(self, collection):
         """
         Add a (free-text) collection of the digital document
         """
-        profile_desc = self.tree.xpath('//tei:profileDesc', namespaces=ns)[0]
-        creation = etree.SubElement(profile_desc, "%screation" % TEI)
-        creation.text = collection
+        profile_desc = self.tree.xpath('//tei:msDesc/tei:msIdentifier', namespaces=ns)[0]
+        coll = etree.SubElement(profile_desc, "%scollection" % TEI)
+        coll.text = collection
```

Comment:
Did you check whether this is DTABf? I **think**, I used this more abstract solution since `collection` of the digital work is not necessarily collection of the original.

Reply:
No, it's not in DTABf at all – sry. But neither is `creation` (and it seems like a misfit). 

You're right in that it's not clear whether `relatedItem/@type=series` applies to the physical copy or digital presentation (which could perhaps be differentiated on the TEI side by `msIdentifier` vs `objectIdentifier` IIUC).

Perhaps the whole thing should rather enter [`biblFull/seriesStmt`](https://www.deutschestextarchiv.de/doku/basisformat/mdSdSeriesStmt.html)?

Reply:
> You're right in that it's not clear whether `relatedItem/@type=series` applies to the physical copy or digital presentation (which could perhaps be differentiated on the TEI side by `msIdentifier` vs `objectIdentifier` IIUC).
> 
> Perhaps the whole thing should rather enter [`biblFull/seriesStmt`](https://www.deutschestextarchiv.de/doku/basisformat/mdSdSeriesStmt.html)?

I am now certain that's the right place. And to differentiate between series of physical copy and series of digital presentation, we could use `fileDesc/sourceDesc/biblFull/seriesStmt`  vs. `fileDesc/seriesStmt` (which is also allowed by DTABf RNG, but not documented).

Let's discuss further under #44!